### DISCUSSION TITLE
Please provide appimage

### DISCUSSION THREAD

**Discussion by eugenialoli** [johnfactotum/foliate#1373] [General]
Hi,
plz provide an appimage for linux in addition to snap/flatpaks. I rather have to deal with a single file that i can delete/update myself rather than installing massive amounts of data each time.

**Comment by johnfactotum** [comment]
I don't currently plan on making an AppImage, though contributions are welcome.

For now, if you don't want to install anything, you might consider
- [Executing Foliate's source code without installing](https://github.com/johnfactotum/foliate?tab=readme-ov-file#run-without-building-or-installing) (Foliate is a JavaScript app, so it's really just a script that you can run with GJS)
- Using an online e-book viewer such as https://johnfactotum.github.io/foliate-js/reader.html.

**Comment by rykel** [comment]
Hi John, I am an avid Linux user and I run mainly AppImages, because I think that it is the way to go for all Linux distros in the future. Then, I came across Foliate when I needed a lightweight but yet capable ebook reader. I understand that there is no official Foliate AppImage, and I do hope that you will do one for us in the future. If I need to buy you are cup of coffee for an official AppImage, I would do it. haha... thanks for reading!
